datasets/blbooksgenre/README.md

```diff
@@ -0,0 +1,389 @@
+ 'annotated': True,
+ 'Type of resource': 0,
+ 'created_at': datetime.datetime(2020, 8, 11, 14, 30, 33),
+ 'annotator_genre': 0}
+```
+
+### Data Fields
+
+The data fields differ slightly between configs. All possible fields for the `annotated_raw` config are listed below. For the `raw` version of the dataset datatypes are usually string to avoid errors when processing missing values.
+
+- `BL record ID`: an internal ID used by the British Library, this can be useful for linking this data to other BL collections.
+- `Name`: name associated with the item (usually author)
+- `Dates associated with name`: dates associated with above e.g. DOB
+- `Type of name`: whether `Name` is a person or an organization etc. 
+- `Role`: i.e. whether `Name` is `author`, `publisher` etc.
+- `All names`: a fuller list of names associated with the item.
+- `Title`: The title of the work
+- `Variant titles`
+- `Series title`
+- `Number within series`
+- `Country of publication`: encoded as a list of countries listed in the metadata
+- `Place of publication`:  encoded as a list of places listed in the metadata 
+- `Publisher`
+- `Date of publication`: this is encoded as a string since this field can include data ranges i.e.`1850-1855`. 
+- `Edition`
+- `Physical description`: encoded as a string since the format of this field varies
+- `Dewey classification`
+- `BL shelfmark`: a British Library shelf mark
+- `Topics`: topics included in the catalogue record
+- `Genre` the genre information included in the original catalogue record note that this is often missing 
+- Languages`; encoded as a list of languages 
+- `Notes`: notes from the catalogue record
+- `BL record ID for physical resource`
+
+The following fields are all generated via the crowdsourcing task (discussed in more detail below)
+
+- `classification_id`: ID for the classification in the annotation task
+- `user_id` ID for the annotator
+- `subject_ids`: internal annotation task ID
+- `annotator_date_pub`: an updated publication data
+- `annotator_normalised_date_pub`: normalized version of the above
+- `annotator_edition_statement` updated edition 
+- `annotator_FAST_genre_terms`: [FAST classification genre terms](https://www.oclc.org/research/areas/data-science/fast.html)
+- `annotator_FAST_subject_terms`: [FAST subject terms](https://www.oclc.org/research/areas/data-science/fast.html)
+- `annotator_comments`: free form comments 
+- `annotator_main_language`
+- `annotator_other_languages_summaries` 
+- `'annotator_summaries_language`
+- `annotator_translation`
+- `annotator_original_language`
+- `annotator_publisher`
+- `annotator_place_pub`
+- `annotator_country`
+- `annotator_title`
+- `Link to digitised book`
+- `annotated`: `bool` flag to indicate if row has annotations or not
+- `created_at`: when the annotation was created
+- `annotator_genre`: the updated annotation for the `genre` of the book.
+
+[More Information Needed]
```

Comment:
Maybe also mention the `label` field of the `title_genre_classifiction` config ?

```suggestion
Finally the `label` field of the `title_genre_classifiction` configuration  is a class label with values 0 (Fiction) or 1 (Non-fiction).

[More Information Needed]
```